src/api/auth/src/authentication.js

```diff
@@ -107,17 +107,43 @@ const strategy = new SamlStrategy(
      *   "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress": "first.last@email.com"
      *   "email": "first.last@email.com"
      * }
+     *
+     * We only really care about the emailaddress claim, which we also use in the Users service.
      */
+    const email =
+      senecaProfile['http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress'];
```

Comment:
How does this link work? Should we add a comment to link to some docs about it?

Reply:
You can read about the schema if you want more info http://docs.oasis-open.org/imi/ns/identity-200810

Reply:
2010, wow...